lib/vtex/client.rb

```diff
@@ -21,7 +21,8 @@ def connection
 
     def self.resources
       {
-        products: ProductResource
+        products: ProductResource,
+        categories: CategoryAndBrandResource
```

Comment:
EH? ... jeje a misscommunication here...

My question was, should this hash needs to be:

```
{
  products: ProductResource,
  categories_and_brands: CategoryAndBrandResource
}
```

or the way it is right now ???